lib/lambda.js

```diff
@@ -135,17 +134,19 @@ module.exports.routeRequest = async (event) => {
 }
 
 async function prepareChrome(opts) {
-  const ChromeWrapper = require('./chrome_wrapper').default
-  wrapper = new ChromeWrapper()
-
-  // Start chrome and fetch the manifest in parallel
-  console.log('Starting chrome')
   const manifest = await getManifest(opts.sessionId)
-  await wrapper.launchLambda()
-
   // We require a manifest in lambda
   if (!manifest) throw new Error(`Missing manifest for ${opts.sessionId}`)
 
+  if (!wrapper) {
+    console.log('Starting chrome')
+    const ChromeWrapper = require('./chrome_wrapper').default
+    wrapper = new ChromeWrapper()
+    await wrapper.launchLambda()
+  } else {
+    console.log("Chrome is already running!")
```

Comment:
I can't comment on that line directly but maybe the comment on line 2 (`// store chrome wrapper globally. In theory we could reuse this between runs`) can be updated?